library/Icingadb/ProvidedHook/Reporting/HostSlaReport.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/* Icinga DB Web | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\ProvidedHook\Reporting;
+
+use Icinga\Application\Icinga;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Reporting\ReportData;
+use Icinga\Module\Reporting\ReportRow;
+use Icinga\Module\Reporting\Timerange;
+use ipl\Sql\Expression;
+use ipl\Stdlib\Filter\Rule;
+
+use function ipl\I18n\t;
+
+class HostSlaReport extends SlaReport
+{
+    public function getName()
+    {
+        $name = t('Host SLA Report');
+        if (Icinga::app()->getModuleManager()->hasEnabled('monitoring')) {
```

Comment:
I feel better knowing this checks for the `idoreports` module. One could still have it enabled despite having monitoring disabled. Then there are two `Host SLA Report` entries.